src/sentry/models/groupinbox.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from enum import Enum
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, JSONField
+
+
+class GroupInboxReason(Enum):
+    NEW = 0
+    UNIGNORED = 1
+    REGRESSION = 2
+
+
+class GroupInbox(Model):
+    """
+    A Group that is in the inbox.
+    """
+
+    __core__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True)
```

Comment:
You'll need to set `db_constraint=False` here. This stops us creating a db level fk constraint on group. We want to avoid this because adding a constraint to group requires us to try and acquire a lock on the group table, which is hard to do since it's so busy.

Reply:
Ah, okay. Added that in :)